src/Reflection/Php/PhpFunctionReflection.php

```diff
@@ -45,16 +41,14 @@ public function __construct(
 		private InitializerExprTypeResolver $initializerExprTypeResolver,
 		private ReflectionFunction $reflection,
 		private Parser $parser,
-		private FunctionCallStatementFinder $functionCallStatementFinder,
-		private Cache $cache,
 		private TemplateTypeMap $templateTypeMap,
 		private array $phpDocParameterTypes,
 		private ?Type $phpDocReturnType,
 		private ?Type $phpDocThrowType,
 		private ?string $deprecatedDescription,
 		private bool $isDeprecated,
 		private bool $isInternal,
-		private bool $isFinal,
+		private bool $isFinal, // @phpstan-ignore property.onlyWritten
```

Comment:
interessting that this PHPStan error shows up with this PR, because the propery was already only-written before this PR